website/docs/terms/ddl.md

```diff
@@ -0,0 +1,126 @@
+
+:::note Note
+Data folks don’t typically write DDL [since dbt will do it for them](https://docs.getdbt.com/docs/about/overview#:~:text=dbt%20allows%20analysts%20avoid%20writing,dbt%20takes%20care%20of%20materialization.).
+:::
+
+To be honest, DDL is definitely some of the drier content that exists out there in the greater data world. However, because DDL commands are often uncompromising and should be used with caution, it’s incredibly important to understand how they work and when they should be used. We hope you can use this page to learn about the basics, strengths, and limitations of DDL statements.
+
+## Types of DDL Statements
+
+![Image of a One Does Not Simply Walk Into Mordor meme prompt that reads: One does not simply drop a table](/img/docs/terms/ddl/drop-a-table-meme.png)
+
+DDL statements are used to create, drop, and manipulate objects in your database. They are often, but not always, unforgiving and irreversible. “With great power comes great responsibility,” is usually the first thing I think of before I execute a DDL command. We’ll highlight some of the primary DDL commands that are used by analytics engineers below.
+
+:::important Important
+The syntax for DDL commands can be pretty database-specific. We are trying to make this glossary page as generic as possible, but please use the “Further Reading” section to see the specifics on how the following DDL commands would be implemented in your database of interest!
+:::
+
+### ALTER
+
+Using the `ALTER` DDL command, you can change an object in your database that already exists. By "change", we specifically mean you can:
+
+- Add new, remove, and rename columns to views and tables
+- Rename a view or table
+- Modify the structure of a view or table
+- And more!
+
+The generic syntax to use the ALTER command is as follows:
+
+```sql
+ALTER <database object type> <database object name>;
+```
+
+To alter a table’s column, you may do something like this:
+
+```sql
+ALTER TABLE customers rename column last_name as last_initial;
+```
+
+In this example, you have to rename the `last_name` column [in jaffle_shop’s](https://github.com/dbt-labs/jaffle_shop) `customers` table to be called `last_initial`.
+
+### DROP
+
+The `DROP` command. Probably the most high-stakes DDL statement one can execute. One that should be used with the *utmost* of care. At its core, an executed `DROP` statement will remove that object from the data warehouse. You can DROP tables, views, schemas, databases, users, functions, and more.
+
+Some data warehouses such as Snowflake allow you to add restrictions to DROP statements to caution you about the impact of dropping a table, view, or schema before it’s actually dropped. In practice, we recommend you never drop raw source tables as they are often your baseline of truth. Your database user also usually needs the correct permissions to drop database objects.
```

Comment:
make this DROP code `DROP`

Reply:
Good catch. Fixed this in the latest commit